.buildconfig.yml

```diff
@@ -14,6 +14,7 @@ projects:
     path: samples/android/app
     artifactId: glean-sample-app
     description: 'An app demoing how to use the Glean library to collect and send telemetry data.'
+    publications: []
```

Comment:
What is this doing? Making sure we never end up publishing the artifacts? If so, would you kindly add a comment? I was not able to find the reference syntax from this file, I'm assuming this is Mozilla-specific stuff?

Reply:
The taskcluster decision task now parses this file to find things.
It looks for a `publications` item and explodes if that is missing.
I could have also fixed the code over there to handle it more gracefully but I also wanted to keep it as close as possible to a-s' config, so it's easier to compare.

Reply:
I see. Consider just adding a note saying that and pointing to any issue filed for taskcluster, if avaialble.